lib/krumo/class.krumo.php

```diff
@@ -1110,7 +1110,7 @@
       <strong class="krumo-class"><?php echo get_class($data);?></strong>
   </div>
 
-  <?php if (count($data)) {
+  <?php {
```

Comment:
Should we remove the curly brackets and make this one line?

Reply:
...also makes perfect sense. Also done 🙂